### PR TITLE
refactored segment dummy changer

### LINKED ISSUE #43: refactor dummy segment changer
- [x]  vectorize compute methods
- [x] clean up magic numbers
- [x] clean up apply method

### PR DESCRIPTION
**Pull request by KueblerJelle**
fixes #43

### CODE REVIEW

File: german_freeval/macro/dummy_segment_changer.py [hunk 1]
```diff
@@ -4,59 +4,54 @@
 
 
 class DummySegmentChanger(SegmentChanger):
+
+    VEHICLES_PER_LANE_PER_KM = 100
+    CAPACITY_PER_LANE = 2000
```
Comment (frunika) [inline german_freeval/macro/dummy_segment_changer.py]: _PER_HOUR

File: german_freeval/macro/dummy_segment_changer.py [hunk 2]
```diff
@@ -4,59 +4,54 @@
...
 
-    def compute_demand_ramp(self, segment: Merging, period: int):
+    def compute_demand_ramp(self, segment: Merging):
         incoming: Source = segment.ramp_in
-        return incoming.demand[period]
+        segment.demand_onramp = PropertyBuilder(
+            name="demand_onramp",
+            type="float",
+            initial_values=incoming.demand.values.copy(),
+        )
```
Comment (frunika) [inline german_freeval/macro/dummy_segment_changer.py]: .build() missing?

**Review by frunika — COMMENTED** [review]
I have two inline comments to solve :)

File: german_freeval/macro/dummy_segment_changer.py (hunk 1 above)
Comment (KueblerJelle) [reply german_freeval/macro/dummy_segment_changer.py]: fixed

File: german_freeval/macro/dummy_segment_changer.py (hunk 2 above)
Comment (KueblerJelle) [reply german_freeval/macro/dummy_segment_changer.py]: fixed

**Review by frunika — APPROVED** [review]

**Pull request merged**